### ISSUE TITLE
Editing / managing export templates in Tome

### ISSUE THREAD

**Issue by pgrimsehl** [npruehs/tome-editor#67]
Current state:

- For each new export template a new set of the required files must be manually created, opened, edited and changed
- The project file must be changed manually to add an export template to the project

It would be nice to manage / create export templates within Tome.

Create / Edit Export Template window, featuring
- Creating / updating all necessary files of the export template file set
- Presenting each file content in a separate edit box, grouped by type ( e.g. Records, Fields, Lists, Maps having their data and delimiter strings grouped, maybe a tab for each group)
- A live preview window showing the resulting export using sample data

Manage Project Export Template window, featuring
- Adding / removing export templates to / from the project
- Changing the export flags for a template

**Comment by npruehs** [comment]
I totally agree, and I love the idea of the live preview. I already have something in mind, similar to live previews for code cleanup in Eclipse. This will be added as soon as the "basic" project settings window #16 has been implemented.
